File: db4oj.tests/src/com/db4o/test/legacy/soda/classes/typedhierarchy/STETH4.java
/* Copyright (C) 2004   Versant Inc.   http://www.db4o.com */

package com.db4o.test.legacy.soda.classes.typedhierarchy;

public class STETH4 extends STETH2{
	
	String foo4;
	
	public STETH4(){
	}
	
	public STETH4(String str1, String str2, String str3){
		super(str1, str2);
		foo4 = str3;
	}
	
}

File: db4o.net/Db4objects.Db4o.Tests/Db4objects.Db4o.Tests/Common/Soda/Classes/Typedhierarchy/STETH4.cs
/* Copyright (C) 2004 - 2011  Versant Inc.  http://www.db4o.com */

using Db4objects.Db4o.Tests.Common.Soda.Classes.Typedhierarchy;

namespace Db4objects.Db4o.Tests.Common.Soda.Classes.Typedhierarchy
{
	public class STETH4 : STETH2
	{
		public string foo4;

		public STETH4()
		{
		}

		public STETH4(string str1, string str2, string str3) : base(str1, str2)
		{
			foo4 = str3;
		}
	}
}
